doc/docs.xml

```diff
@@ -288,17 +288,13 @@
 	<refsect1>
 		<title>See Also</title>
 		<para>
-			<ulink url="http://conky.sourceforge.net/">
+			<ulink url="https://github.com/brndnmtthws/conky">
 				http://conky.sourceforge.net/</ulink>
```

Comment:
Should should probably be `https://github.com/brndnmtthws/conky` since you're in there anyway.